### ISSUE TITLE
outputCheck failed to run

### ISSUE THREAD

**Issue by hd-deman** [sensu-plugins/sensu-plugins-docker#5]
```
command: 
check-container.rb --host tcp://172.16.0.3:2375/ mobile_es_1

output: 
Check failed to run: uninitialized constant Docker::Error, ["/opt/rbenv/versions/2.0.0-p598/lib/ruby/gems/2.0.0/gems/sensu-plugins-docker-0.0.2/bin/check-container.rb:62:in `rescue in run'", "/opt/rbenv/versions/2.0.0-p598/lib/ruby/gems/2.0.0/gems/sensu-plugins-docker-0.0.2/bin/check-container.rb:52:in `run'", "/opt/rbenv/versions/2.0.0-p598/lib/ruby/gems/2.0.0/gems/sensu-plugin-1.1.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"] 
```
Any ideas why it's happening?

**Comment by hd-deman** [comment]
This happening because on rubygems 
```
RUNTIME DEPENDENCIES:
docker = 0.0.1
```
But current version use  'docker-api',   '~> 1.21`

**Comment by hd-deman** [comment]
@sstarcher ?

**Comment by sstarcher** [comment]
@hd-deman I would recommend removing the rubygem docker and installing docker-api 1.20.0 or greater

**Comment by soichih** [comment]
+1

**Comment by kelcecil** [comment]
@sstarcher I reworked this completely. I removed all Docker gems and made an HTTP call straight to either the HTTP endpoint or the Docker socket.

**Comment by sstarcher** [comment]
@kelcecil you going to share the code ;)

**Comment by eheydrick** [comment]
@kelcecil's changes have been released in 1.0.0. Can you give that a shot and provide feedback.
